index.bs

```diff
@@ -753,6 +753,18 @@ To <dfn>find a matching trigger spec</dfn> given an [=attribution source=]
 
     </dl>
 
+<h3 id="destination-limit-algorithm-header">Destination limit algorithm</h3>
+
+A <dfn>destination limit algorithm</dfn> is one of the following:
+
+<dl dfn-for="destination limit algorithm">
+: "<dfn><code>priority_fifo</code></dfn>"
+:: Counts the highest prioritized and most recent [=attribution source/attribution destinations=].
```

Comment:
Similar to my comment in `EVENT.md`, I find `counts...` to be hard to understand. IIUC all destinations are *counted*, but the algorithm and priority are used to determine which ones to evict.